libs/barista-components/stacked-series-chart/src/stacked-series-chart.ts

```diff
@@ -0,0 +1,434 @@
+    return this._fillMode;
+  }
+  set fillMode(value: DtStackedSeriesChartFillMode) {
+    if (value !== this._fillMode) {
+      this._fillMode = value ?? 'relative';
+      this._render();
+    }
+  }
+  private _fillMode: DtStackedSeriesChartFillMode = 'relative';
+
+  /**
+   * Sets the display mode for the stacked-series-chart values
+   * in legend to either 'none' 'percent' or 'absolute'.
+   * Only valid for single track chart.
+   */
+  @Input() valueDisplayMode: DtStackedSeriesChartValueDisplayMode = 'none';
+  /** @internal Will be true if display mode is not 'none' and only has one series */
+  _canShowValue: boolean;
+
+  /** Array of legends that can be used to toggle bar nodes. Useful when the legend used is outside this component */
+  @Input()
+  get legends(): DtStackedSeriesChartLegend[] | undefined {
+    return this._legends ?? [];
+  }
+  set legends(value: DtStackedSeriesChartLegend[] | undefined) {
+    if (value !== this._legends) {
+      this._legends =
+        value && value.length > 0 ? value : getLegends(this._series);
+
+      updateNodesVisibility(this._filledSeries, this._legends);
+
+      this._render();
+    }
+  }
+  _legends: DtStackedSeriesChartLegend[];
+
+  /** Visibility of the legend */
+  @Input() visibleLegend: boolean = true;
+
+  /** Whether background should be transparent or show a background. Default: true */
+  @Input() visibleTrackBackground: boolean = true;
+
+  /** Visibility of series label */
+  @Input() visibleLabel: boolean = true;
+
+  /** Display mode */
+  @Input()
+  get mode(): DtStackedSeriesChartMode {
+    return this._mode;
+  }
+  set mode(value: DtStackedSeriesChartMode) {
+    if (this._mode !== value) {
+      this._mode = value ?? 'bar';
+      // as template changes and we rely on dimensions we have to use a lifecycle hook
+      this._shouldUpdateTicks.next();
+    }
+  }
+  _mode: DtStackedSeriesChartMode = 'bar';
+
+  /** Maximum size of the track */
+  @Input() maxTrackSize: number = 16;
+
+  /** @internal Reference to the root svgElement. */
+  @ViewChild('valueAxis') _valueAxis;
+
+  /** Visibility of value axis */
+  @Input() visibleValueAxis: boolean = true;
+
+  /** @internal Ticks for value axis */
+  _axisTicks: { pos: number; value: number; valueRelative: number }[] = [];
+
+  /** @internal Value axis width to allow it inside the boundaries of the component */
+  _valueAxisSize: { absolute: number; relative: number } = {
+    absolute: 0,
+    relative: 0,
+  };
+
+  /** Indicates when ticks should be recalculated */
+  private _shouldUpdateTicks = new Subject();
+
+  /** Subject to be called upon component destroy to remove pending subscriptions */
+  private readonly _destroy$ = new Subject<void>();
+
+  constructor(
+    private readonly _changeDetectorRef: ChangeDetectorRef,
+    private _resizer: DtViewportResizer,
+    private _zone: NgZone,
+    private _overlayService: DtOverlay,
+    // TODO: remove this sanitizer when ivy is no longer opt out
```

Comment:
Can we deprecate this and put the deprecation warning inside the message?
@ffriedl89 what do you think about this? (I think with Angular 11 we can drop it then.)

```suggestion
    /**
     * @deprecated Remove the sanitizer when we don't have to support ivy anymore.
     * @breaking-change Remove the DomSanitizer. (Version: TBD)
     */
```